src/test/javascript/spec/screenshot.ts

```diff
@@ -116,4 +131,159 @@ describe('Screenshot Tests', () => {
     const reviewCompleteButton = await $('button[data-testid="review-complete-button"]');
     await reviewCompleteButton.click();
   });
+
+  it('should compare add mutation modal', async () => {
+    await browser.url(`${BASE_URL}/curation/EMPTYGENE/somatic`);
+
+    // Click to open mutation modal
+    const addMutationButton = await $('button=Add Mutation');
+    await addMutationButton.click();
+
+    // Add a new mutation
+    const mutationNameInput = await $('input#add-mutation-modal-input');
+    await mutationNameInput.setValue('V600E');
+    await browser.keys('Enter');
+
+    const addMutationModal = await $('div[id="default-add-mutation-modal"]');
+    await addMutationModal.waitForDisplayed();
+
+    const result = await browser.checkElement(addMutationModal, 'add-mutation-modal', methodOptions);
+    expect(result).toBeLessThanOrEqual(ALLOWED_MISMATCH_PERCENTAGE);
+  });
+
+  it('should compare gene type on review page', async () => {
+    await browser.url(`${BASE_URL}/curation/EMPTYGENE/somatic`);
+
+    const tsgCheckbox = await $("input[id='Genes/EMPTYGENE/type/tsg']");
+    await tsgCheckbox.waitForDisplayed();
+    await tsgCheckbox.click();
+
+    const reviewButton = await $('button[data-testid="review-button"]');
+    await reviewButton.click();
+    const reviewCollapsible = await $('div[data-testid="collapsible"]');
+    await reviewCollapsible.waitForDisplayed();
+
+    const result = await browser.checkElement(reviewCollapsible, 'gene-type-review-collapsible', methodOptions);
+    expect(result).toBeLessThanOrEqual(ALLOWED_MISMATCH_PERCENTAGE);
+  });
+
+  it('should compare updated gene summary on review page', async () => {
+    await browser.url(`${BASE_URL}/curation/TP53/somatic`);
+
+    const mutationEffectDesc = await $("textarea[id='Genes/TP53/summary']");
+    await mutationEffectDesc.waitForDisplayed();
+    await mutationEffectDesc.setValue('TP53 is the most frequently mutated gene in cancer. Additional text.');
+
+    // Go to review page to compare collapsible
+    const reviewButton = await $('button[data-testid="review-button"]');
+    await reviewButton.click();
+    const reviewCollapsible = await $('div[data-testid="collapsible"]');
+    await reviewCollapsible.waitForDisplayed();
+
+    const result = await browser.checkElement(reviewCollapsible, 'updated-gene-summary-review-collapsible', methodOptions);
+    expect(result).toBeLessThanOrEqual(ALLOWED_MISMATCH_PERCENTAGE);
```

Comment:
Is this gonna be our default for checking all screenshots? Maybe create our own expect method?